app/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,37 @@
+<!--
+  ~ Copyright (c) 2018 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
```

Comment:
For me, every line in AS is flagging it up red as an error. is this just a tooling issue to be ignored?

<img width="1371" alt="screen shot 2018-09-21 at 11 31 37" src="https://user-images.githubusercontent.com/1336281/45876402-7cb65880-bd92-11e8-9e4c-3da5169e374b.png">

Reply:
Looks like a tooling issue, I don't see that